tools/add_triage_signature.py

```diff
@@ -347,6 +348,7 @@ def build_description(self, url, cluster_md):
                         "logs_url": self._logs_url_to_ui(url).strip('/') + '/',
                         "domain": cluster_md['email_domain'],
                         "openshift_version": cluster_md['openshift_version'],
+                        "platform_type": cluster_md['platform']['type'],
```

Comment:
If it's an optional field (judging from code below), then we should probably do:
```suggestion
                        "platform_type": cluster_md.get('platform', {}).get('type', 'N/A'),
```

Reply:
It is not, I expect this field to always exist, but anyway a good suggestion